### PR TITLE
feat(block-reverter): only require private key for sending revert transactions

### PR DESCRIPTION
**Pull request by montekki**
## What ❔

Previosly whenever working with `block_reverter` you needed to specify the private key that is eneded to send the revert transaction. However this key is only required for one operation and makes no sense while the user just reverts a database or a rocksdb. This patch
* Makes operator private key required only for sending revert transactions
* Makes operator _address_ required to print suggested values

Thus limiting the usage of private key to a minimum.

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

**Review by Deniallugo — APPROVED** [review]

File: core/tests/revert-test/tests/revert-and-restart.test.ts [hunk 1]
```diff
@@ -150,7 +150,7 @@ describe('Block reverting test', function () {
     step('revert blocks', async () => {
         const executedProcess = await utils.exec(
             'cd $ZKSYNC_HOME && ' +
-                'RUST_LOG=off cargo run --bin block_reverter --release -- print-suggested-values --json'
+                'RUST_LOG=off cargo run --bin block_reverter --release -- print-suggested-values --json --operator-address 0xde03a0B5963f75f1C8485B355fF6D30f3093BDE7'
```
Comment (Deniallugo) [inline core/tests/revert-test/tests/revert-and-restart.test.ts:153]: Probably it's better to get it from config

Comment (montekki) [reply core/tests/revert-test/tests/revert-and-restart.test.ts:153]: Does the address, not the key exist there ?

Comment (Deniallugo) [reply core/tests/revert-test/tests/revert-and-restart.test.ts:153]: for tests everything exists

Comment (montekki) [reply core/tests/revert-test/tests/revert-and-restart.test.ts]: done, takes it from env now

**Review by Deniallugo — DISMISSED** [review]

**Review by Deniallugo — APPROVED** [review]

**Pull request merged**